shiny/ui/_theme.py

```diff
@@ -7,6 +7,7 @@
 import textwrap
 from typing import Any, Literal, Optional, Sequence, TypeVar
 
+from brand_yml import Brand
```

Comment:
This should be under `if TYPE_CHECKING`, and then import when we actually need it